.github/workflows/python-ci.yml

```diff
@@ -50,12 +50,9 @@ jobs:
           ${{ matrix.flint-inst-cmd }}
       - name: Setup nox
         uses: excitedleigh/setup-nox@v2.1.0
-      - name: Test on 3.7
-        run: nox -s tests-3.7
-      # Python 3.8 and 3.9 are only tested on Linux.
-      - if: runner.os == 'Linux'
-        name: Test on 3.8
+      - name: Test on 3.8
         run: nox -s tests-3.8
+      # Python 3.9 are only tested on Linux.
```

Comment:
Something for another PR (feel free to use this message and convert it into an issue that tracks this):
The configuration here should really be updated to our latest best practices as it does not cover Python 3.11 at all (which came out quite a while ago).
It should be straight-forward to adapt from https://github.com/cda-tum/qcec/blob/main/.github/workflows/python-ci.yml